models/webhook.go

```diff
@@ -572,7 +574,8 @@ type HookTask struct {
 	UUID            string
 	Type            HookTaskType
 	URL             string `xorm:"TEXT"`
-	Signature       string `xorm:"TEXT"`
```

Comment:
We need a database migration

Reply:
Check 4309549f5af1018f64d802e4c6e5aaa13e3d4add

Not sure about the mssql upgrade script. Should work